static/app/views/replays/detail/accessibility/accessibilityTableCell.tsx

```diff
@@ -0,0 +1,148 @@
+};
+
+interface Props extends ReturnType<typeof useCrumbHandlers> {
+  a11yIssue: HydratedA11yIssue;
+  columnIndex: number;
+  currentHoverTime: number | undefined;
+  currentTime: number;
+  onClickCell: (props: {dataIndex: number; rowIndex: number}) => void;
+  rowIndex: number;
+  sortConfig: ReturnType<typeof useSortAccessibility>['sortConfig'];
+  startTimestampMs: number;
+  style: CSSProperties;
+}
+
+const AccessibilityTableCell = forwardRef<HTMLDivElement, Props>(
+  (
+    {
+      a11yIssue,
+      columnIndex,
+      currentHoverTime,
+      currentTime,
+      onClickCell,
+      onClickTimestamp,
+      onMouseEnter,
+      onMouseLeave,
+      rowIndex,
+      sortConfig,
+      startTimestampMs,
+      style,
+    }: Props,
+    ref
+  ) => {
+    // Rows include the sortable header, the dataIndex does not
+    const dataIndex = rowIndex - 1;
+
+    const {getParamValue} = useUrlParams('a_detail_row', '');
+    const isSelected = getParamValue() === String(dataIndex);
+
+    const hasOccurred = currentTime >= a11yIssue.offsetMs;
+    const isBeforeHover =
+      currentHoverTime === undefined || currentHoverTime >= a11yIssue.offsetMs;
+
+    const isByTimestamp = sortConfig.by === 'timestampMs';
+    const isAsc = isByTimestamp ? sortConfig.asc : undefined;
+    const columnProps = {
+      className: classNames({
+        beforeCurrentTime: isByTimestamp
+          ? isAsc
+            ? hasOccurred
+            : !hasOccurred
+          : undefined,
+        afterCurrentTime: isByTimestamp
+          ? isAsc
+            ? !hasOccurred
+            : hasOccurred
+          : undefined,
+        beforeHoverTime:
+          isByTimestamp && currentHoverTime !== undefined
+            ? isAsc
+              ? isBeforeHover
+              : !isBeforeHover
+            : undefined,
+        afterHoverTime:
+          isByTimestamp && currentHoverTime !== undefined
+            ? isAsc
+              ? !isBeforeHover
+              : isBeforeHover
+            : undefined,
+      }),
+      hasOccurred: isByTimestamp ? hasOccurred : undefined,
+      isSelected,
+      onClick: () => onClickCell({dataIndex, rowIndex}),
+      onMouseEnter: () => onMouseEnter(a11yIssue),
+      onMouseLeave: () => onMouseLeave(a11yIssue),
+      ref,
+      style,
+    } as ComponentProps<typeof Cell>;
+
+    const renderFns = [
+      () => (
+        <Cell {...columnProps}>
+          <Text>
+            {a11yIssue.impact ? (
+              <Tooltip title={a11yIssue.impact ?? EMPTY_CELL}>
+                {IMPACT_ICON_MAPPING[a11yIssue.impact]}
+              </Tooltip>
+            ) : (
+              EMPTY_CELL
+            )}
+          </Text>
+        </Cell>
+      ),
+      () => (
+        <Cell {...columnProps}>
+          <Text>{a11yIssue.id ?? EMPTY_CELL}</Text>
+        </Cell>
+      ),
+      () => (
+        <Cell {...columnProps}>
+          <Text>{a11yIssue.elements?.[0].element ?? EMPTY_CELL}</Text>
```

Comment:
Is there a `<code>`-type component we can use here?

Reply:
oh yeah

![SCR-20231012-jttt](https://github.com/getsentry/sentry/assets/187460/6f705241-7bdf-4b0a-8243-a8eab5e1e3de)